pkg/plugins/runtime/gateway/testdata/http/18-gateway-route.yaml

```diff
@@ -1,13 +1,13 @@
 Clusters:
   Resources:
-    external-httpbin-4a2cc28cd5e983f1:
+    external-httpbin-2528f53d03636b9d:
```

Comment:
why did it change?

Reply:
The resource hash is computed from its body. As the timeout changed the hash changed too